WalletWasabi/Helpers/EnvironmentHelpers.cs

```diff
@@ -45,5 +46,31 @@ public static string GetDataDir(string appName)
 
 			return directory;
 		}
+
+		/// <summary>
+		/// Executes a command with bash.
+		/// https://stackoverflow.com/a/47918132/2061103
+		/// </summary>
+		/// <param name="cmd"></param>
+		public static void BashExec(string cmd)
+		{
+			var escapedArgs = cmd.Replace("\"", "\\\"");
+
+			var process = new Process
+			{
+				StartInfo = new ProcessStartInfo
+				{
+					RedirectStandardOutput = true,
+					UseShellExecute = false,
+					CreateNoWindow = true,
+					WindowStyle = ProcessWindowStyle.Hidden,
+					FileName = "/bin/bash",
```

Comment:
Use `sh` instead of `bash` because if well it is in the 99% of the distros, we shouldn't assume it is available in the 100% of the cases.